### PR TITLE
Update `/post/create` with image upload

### LINKED ISSUE #48: Update `/post/create` with Image Upload schema
Similar to `/user/create/`, but it wouldn't be Optional anymore.

### PR DESCRIPTION
**Pull request by aitikgupta**
Resolves #48

### CODE REVIEW

File: backend/app/server/routes/post.py [hunk 1]
```diff
@@ -9,18 +10,23 @@
     update_post,
     report_post
 )
+from ..controllers.upload import upload_image
 from ..models.post import (
     ResponseModel,
-    PostSchema,
```
Comment (bodhisha) [inline backend/app/server/routes/post.py:14]: We don't need the PostSchema and UserSchema in models also, right?

**Review by bodhisha — CHANGES_REQUESTED** [review]

File: backend/app/server/routes/post.py (hunk 1 above)
Comment (aitikgupta) [reply backend/app/server/routes/post.py:14]: Nope, which is why I removed it.

Comment (aitikgupta) [reply backend/app/server/routes/post.py:14]: But we can keep the whole schema as a reference in the `models/`

Comment (bodhisha) [reply backend/app/server/routes/post.py:14]: Yup.. Cool 🎉

**Review by bodhisha — APPROVED** [review]
Looks Good🎉

**Pull request merged**